resources/capability-statement.json

```diff
@@ -0,0 +1,117 @@
+{
+  "resourceType": "CapabilityStatement",
+  "id": "bulk-data",
+  "url": "http://hl7.org/fhir/us/core/CapabilityStatement/bulk-data",
```

Comment:
I'm not sure about this URL -- I would have expected something like `http://hl7.org/fhir/uv/bulk-data/CapabilityStatement/bulk-data`... but unclear to me what the conventions are. @grahamegrieve can you clarify? Or @lmckenzi ?

Reply:
I think we want conformance resources like:

http://hl7.org/fhir/bulk-data/CapabilityStatement/bulk-data
http://hl7.org/fhir/bulk-data/ImplementationGuide/bulk-data
http://hl7.org/fhir/bulk-data/OperationDefinition/export
http://hl7.org/fhir/bulk-data/OperationDefinition/patient-export
http://hl7.org/fhir/bulk-data/OperationDefinition/group-export

Reply:
@ricky1uphealth 
- [ ] capability statement displayed in the ig, use that url within this json 
- [ ] ig resource displayed in the ig, use that url within this json